src/apiCall.js

```diff
@@ -14,38 +30,39 @@ module.exports = function apiCall(file, apiPort) {
   const { paths, ...remainingSpec } = openSpec
   const operationIds = {}
 
-  for (const route of Object.keys(paths)) {
+  Object.keys(paths).forEach((route) => {
```

Comment:
Could also do 
```suggestion
  Object.values(paths).forEach((actions) => {
```
Minor point - then you don't need the next line to index into `paths`